guides/setup-new-package.md

```diff
@@ -0,0 +1,40 @@
+# Setup a New Package for Pulsar Cooperative
+
+This guide will walk you through the steps needed to configure and setup a brand new forked package on Pulsar Cooperative.
+
+## Renaming the Package/Repository
+
+Since the original name of the package is reserved on the Pulsar Package Registry, it will need to be renamed, both within the `package.json` as well as the repository name itself (while the name doesn't have to change, it's best practice to keep these names in sync).
+
+When renaming the package the loose convention would be to reuse the original package name, but adding `-pulsar` to the end, such as `[original-name]-pulsar` e.g. `language-cobol-pulsar`.
+
+## Configuring the Repository
+
+This repository has several helpful template resources to aide in the initial setup, these resources are available within `./package-template` and show the text that should be added to a package's readme, as well as many of the actions that will be needed.
+
+### Configuring Repository Settings
+
+This setup relates to settings that must be changed on the repository itself, via the GitHub website.
+
+* Add the following values to the repository's secrets/variables:
+
+  - `PPR_AUTH`: This should be set at the organization level, and does not need to be set per repository. This should be a Pulsar Package Registry API key, that allows access to the PPR under this organizations name.
+  - `PUBLISHED`: This is a value that represents a boolean (although it's still just text) that determines if the package has been published to the PPR already. Using this allows us to ensure that automatic version updates aren't done prior to the package being ready, and ensures we don't accidentally try to publish it twice. Prior to the package being published. Don't set this value manually, as it's lack of existence works just as well as creating it manually and setting it to `false`. Otherwise the `publish.yml` GitHub workflow will create this variable as `true` once run.
```

Comment:
```suggestion
  - `PUBLISHED`: This is a value that represents a boolean (although it's still just text) that determines if the package has been published to the PPR already. Using this allows us to ensure that automatic version updates aren't done prior to the package being ready, and ensures we don't accidentally try to publish it twice. This value should not be set manually prior to the package being published, as its lack of existence works just as well as creating it manually and setting it to `false`. Otherwise the `publish.yml` GitHub workflow will create this variable as `true` once run.
```